setup.py

```diff
@@ -38,7 +38,7 @@
 from setuptools.command import sdist as setuptools_sdist
 
 
-CYTHON_DEPENDENCY = 'Cython(>=0.29.24,<0.30.0)'
+CYTHON_DEPENDENCY = 'Cython(>=3.0.11,<3.1.0)'
```

Comment:
Please leave the lower bound on 0.29.x before every other Cython package is upgraded (most importantly, server).  This is for metapkg reasons.

Reply:
0.29.x doesn't support 3.13, AFAICT